test_scripts/API/VehicleData/OnVehicleData/005_OnVD_App_is_not_subscribed.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL doesn't transfer OnVehicleData notification with <vd_param> parameter to App
+-- if an app is not subscribed to this parameter
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) SubscribeVehicleData, UnsubscribeVehicleData, OnVehicleData RPCs and
+--  <vd_param> parameter are allowed by policies
+-- 3) App is registered
+-- 4) App is subscribed to <vd_param> parameter
+--
+-- In case:
+-- 1) HMI sends valid OnVehicleData notification with <vd_param> parameter data to SDL
+-- SDL does:
+-- - a) transfer this notification to App
+-- 2) App unsubscribes from <vd_param> parameter
```

Comment:
In my opinion, this case is more related to unsubscribe case, in this case the subscription can be checked before subscribeVD request and after